<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tuner_Window</class>
 <widget class="QDialog" name="Tuner_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>441</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="essentialTab">
    <attribute name="title">
     <string>Essential</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="generalTabLayout" stretch="1">
      <item>
       <widget class="QGroupBox" name="imageAdjustGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Image Adjust</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="brightnessContrastLayout">
           <item>
            <layout class="QHBoxLayout" name="contrastLineLayout">
             <item>
              <widget class="QLabel" name="contrastLabel">
               <property name="text">
                <string>Contrast    </string>
               </property>
               <property name="buddy">
                <cstring>contrastSlider</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="contrastSlider">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="contrastGap">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="brightnessLineLayout">
             <item>
              <widget class="QLabel" name="brightnessLabel">
               <property name="text">
                <string>Brightness</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="brightnessSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="brightnessGap">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="lowPassTab">
    <attribute name="title">
     <string>Low-Pass Filters</string>
    </attribute>
    <widget class="QGroupBox" name="lowPassGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Smoothing Filters</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>399</width>
        <height>200</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="lowPassMainLayout">
       <item>
        <layout class="QVBoxLayout" name="filterTypeRaidoGroupLayout">
         <item>
          <widget class="QRadioButton" name="simpleRadioButton">
           <property name="text">
            <string>Simple Blur</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="medianRadioButton">
           <property name="text">
            <string>Median Blur</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="guassianLayout">
           <item>
            <widget class="QRadioButton" name="guassianRadioButton">
             <property name="text">
              <string>Guass Blur</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="guassianMatchKernelCheckBox">
             <property name="text">
              <string>Match Kernl?</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="bilateralParametersLayout" stretch="0,0,0,0">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QRadioButton" name="bilateralRadioButton">
             <property name="text">
              <string>Bilateral Filter</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sigmaClearLayout">
             <item>
              <widget class="QLabel" name="sigmaClearLabel">
               <property name="text">
                <string>    sigma clear  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sigmaClearLineEdit">
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sigmaSpaceLayout">
             <item>
              <widget class="QLabel" name="sigmaSpaceLabel">
               <property name="text">
                <string>    sigma space</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sigmaSpaceLineEdit">
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="bilateralDiameterLayout">
             <item>
              <widget class="QLabel" name="bilateralDiameterLabel">
               <property name="text">
                <string>    depth           </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="bilateralDiameterLineEdit">
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="xKernailSizeLayout">
         <item>
          <widget class="QLabel" name="xKernalSizeTitleLabel">
           <property name="text">
            <string>X-Kernel Size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="xKernalSizeValueLabel">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="xKernalSizeDial">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="yKernalSizeLayout">
         <item>
          <widget class="QLabel" name="xKernalTitleLabel">
           <property name="text">
            <string>Y-Kernal Size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yKernalSizeValueLabel">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="yKernalSizeDial"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="highPassTab">
    <attribute name="title">
     <string>High-Pass Filters</string>
    </attribute>
    <widget class="QGroupBox" name="highPassGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>411</height>
      </rect>
     </property>
     <property name="title">
      <string>Edge Detection</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>391</width>
        <height>290</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="highPassMainLayout">
       <item>
        <layout class="QHBoxLayout" name="sobelLayout">
         <item>
          <widget class="QRadioButton" name="sobelRadioButton">
           <property name="text">
            <string>Sobel</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="sobelSlidersLayout">
           <item>
            <layout class="QHBoxLayout" name="sobelXWeightLineLayout">
             <item>
              <widget class="QLabel" name="sobelXWeightLabel">
               <property name="text">
                <string>x weight    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sobelXWeightSlider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sobelYWeightLineLayout">
             <item>
              <widget class="QLabel" name="sobelYWeightLabel">
               <property name="text">
                <string>y weight    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sobelYWeightSlider">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sobelGammaLineLayout">
             <item>
              <widget class="QLabel" name="sobelGammaLabel">
               <property name="text">
                <string>Gamma     </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sobelGammaSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sobelKernalSizeLineLayout">
             <item>
              <widget class="QLabel" name="sobelKernalSizeLabel">
               <property name="text">
                <string>Kernel Size</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sobelKernalSizeSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>7</number>
               </property>
               <property name="singleStep">
                <number>3</number>
               </property>
               <property name="pageStep">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="highPassSobelLaplacianSeperator">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="laplacianLayout">
         <item>
          <widget class="QRadioButton" name="laplacianRadioButton">
           <property name="text">
            <string>Laplacian</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="laplacianSlidersLayout">
           <item>
            <layout class="QHBoxLayout" name="laplacianScaleLineLayout">
             <item>
              <widget class="QLabel" name="laplacianScaleLabel">
               <property name="text">
                <string>Scale        </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="laplacianScaleSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="laplacianKernalLineLayout">
             <item>
              <widget class="QLabel" name="laplacianKernalLabel">
               <property name="text">
                <string>Kernal Size</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="laplacianKernalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="highPassLaplacianCannySeperator">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="morphLayout">
         <item>
          <widget class="QRadioButton" name="morphRadioButton">
           <property name="text">
            <string>Morph</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="morphSlidersLayout">
           <item>
            <layout class="QHBoxLayout" name="morphTypelLayout">
             <item>
              <widget class="QLabel" name="morphTypeLabel">
               <property name="text">
                <string>Type                               </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="morphTypeLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="morphElementLayout">
             <item>
              <widget class="QLabel" name="morphElementLabel">
               <property name="text">
                <string>Element</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="morphElementSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="morphSizeLayout">
             <item>
              <widget class="QLabel" name="morphSizeLabel">
               <property name="text">
                <string>Size</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="morphSizeSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="sliderPosition">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="morphIterationLayout">
             <item>
              <widget class="QLabel" name="morphIterationLabel">
               <property name="text">
                <string>Iterations</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="morphIterationSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>256</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="morphOperationLayout">
             <item>
              <widget class="QLabel" name="morphOperationLabel">
               <property name="text">
                <string>Operation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="morphOperationSlider">
               <property name="minimum">
                <number>3</number>
               </property>
               <property name="maximum">
                <number>7</number>
               </property>
               <property name="singleStep">
                <number>2</number>
               </property>
               <property name="pageStep">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="playerWindow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>34</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tunerTitleLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>8</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Individual Filter Tuning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xKernalSizeDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xKernalSizeValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
